quarkus.ls/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/model/values/quarkus-values-rules.json

```diff
@@ -0,0 +1,105 @@
+				},
+				{
+					"name": "WARNING",
+					"docs": "WARNING is a message level indicating a potential problem.\n\nIn general WARNING messages should describe events that will be of interest to end users or system managers, or which indicate potential problems. This level is initialized to `900`."
+				},
+				{
+					"name": "INFO",
+					"docs": "INFO is a message level for informational messages.\n\nTypically INFO messages will be written to the console or its equivalent. So the INFO level should only be used for reasonably significant messages that will make sense to end users and system administrators. This level is initialized to `800`."
+				},
+				{
+					"name": "CONFIG",
+					"docs": "CONFIG is a message level for static configuration messages.\n\nCONFIG messages are intended to provide a variety of static configuration information, to assist in debugging problems that may be associated with particular configurations. For example, CONFIG message might include the CPU type, the graphics depth, the GUI look-and-feel, etc. This level is initialized to `700`."
+				},
+				{
+					"name": "FINE",
+					"docs": "FINE is a message level providing tracing information.\n\nAll of FINE, FINER, and FINEST are intended for relatively detailed tracing.  The exact meaning of the three levels will vary between subsystems, but in general, FINEST should be used for the most voluminous detailed output, FINER for somewhat less detailed output, and FINE for the  lowest volume (and most important) messages.\n\nIn general the FINE level should be used for information that will be broadly interesting to developers who do not have a specialized interest in the specific subsystem.\n\nFINE messages might include things like minor (recoverable) failures.  Issues indicating potential performance problems are also worth logging as FINE. This level is initialized to `500`."
+				},
+				{
+					"name": "FINER",
+					"docs": "FINER indicates a fairly detailed tracing message. By default logging calls for entering, returning, or throwing an exception are traced at this level. This level is initialized to `400`."
+				},
+				{
+					"name": "FINEST",
+					"docs": "FINEST indicates a highly detailed tracing message. This level is initialized to `300`."
+				},
+				{
+					"name": "ALL",
+					"docs": "ALL indicates that all messages should be logged. This level is initialized to `Integer.MIN_VALUE`."
+				}
+			]
+		}
+	],
+	"rules": [
+		{
+			"matcher": {
+				"types": [
+					"java.util.logging.Level",
+					"java.util.Optional<java.util.logging.Level>"
+				]
+			},
+			"valuesRef": [
+				"#logginglevel"
+			]
+		},
+		{
+			"matcher": {
+				"names": [
+					"quarkus.log.category.{*}.level",
+					"quarkus.log.category.{*}.min-level"

```

Comment:
when applying the templates, we insert:
```
quarkus.log.category.key.level=key
"quarkus.log.category.key.min-level=key
```
I expect the injected values to be the actual default values, or at the very list something from the list of acceptable values

Reply:
Indeed there were a bug with index. Now it should work.